package NLemmerCacheProto;

import "library/cpp/langmask/proto/langmask.proto";
import "library/cpp/token/serialization/protos/char_span.proto";

message TKey {
    required string Word = 1;
    required NProto.TCharSpan Span = 2;
    required uint32 Type = 3;
    required NProto.TLangMask LangMask = 4;
    required bool FilterForms = 5;
};
